Lib/test/test_email/test_message.py

```diff
@@ -958,6 +958,46 @@ def test_folding_with_utf8_encoding_8(self):
                          b'123456789-123456789\n 123456789 Hello '
                          b'=?utf-8?q?W=C3=B6rld!?= 123456789 123456789\n\n')
 
+
+    def test_folding_with_short_nospace_1(self):
+        # bpo-36520
+        #
+        # Fold a line that contains a long whitespace after
+        # the fold point.
+
+        m = EmailMessage(policy.default)
+        m['Message-ID'] = '12345678912345678123456789123456789123456789'
```

Comment:
```suggestion
        m['Message-ID'] = '123456789' * 3
```

(Note that there was a missing 9 in the sequence).